lib/vanagon/cli/dependencies.rb

```diff
@@ -0,0 +1,88 @@
+require 'docopt'
+require 'json'
+require 'vanagon/logger'
+
+class Vanagon
+  class CLI
+    class Dependencies < Vanagon::CLI
+      DOCUMENTATION = <<~DOCOPT.freeze
+        Usage:
+        dependencies [options] <project-name> <platforms>
+        Options:
+          -h, --help                       Display help
+          -c, --configdir DIRECTORY        Configuration directory [default: #{Dir.pwd}/configs]
+          -w, --workdir DIRECTORY          Working directory on the local host
+          -v, --verbose                    Only here for backwards compatibility. Does nothing.
+        Project-Name:
+          May be a project name of a project from the configs/projects directory or 'all' to generate dependencies for all projects.
+        Platforms:
+          May be a platform name of a platform from the configs/platforms directory or 'all' to generate dependencies for all platforms.
+      DOCOPT
+
+      def parse(argv)
+        Docopt.docopt(DOCUMENTATION, { argv: argv })
+      rescue Docopt::Exit => e
+        VanagonLogger.error e.message
+        exit 1
+      end
+
+      def run(options)
+        platforms_directory = File.join(options[:configdir], 'platforms')
+        projects_directory = File.join(options[:configdir], 'projects')
+
+        unless Dir.exist?(projects_directory) && Dir.exist?(platforms_directory)
+          VanagonLogger.error "Path to #{File.join(options[:configdir], 'platforms')} or #{File.join(options[:configdir], 'projects')} not found."
+          exit 1
+        end
+
+        projects = [options[:project_name]]
+        if projects.include?('all')
+          projects = Dir.children(File.join(options[:configdir], 'projects')).map do |project|
```

Comment:
`File.join(options[:configdir], 'projects')` => `projects_directory`

There's a few more, I'm sure it makes sense from here

Reply:
Thanks for these comments and the ones from earlier today. I'll get these fixed tomorrow morning.